qiskit_machine_learning/kernels/quantum_kernel.py

```diff
@@ -223,106 +238,108 @@ def evaluate(self, x_vec: np.ndarray, y_vec: np.ndarray = None) -> np.ndarray:
                     "x_vec and class feature map have incompatible dimensions.\n" +
                     "x_vec has %s dimensions, but feature map has %s." %
                     (x_vec.shape[1], self._feature_map.num_parameters)) from AttributeError
-
         if y_vec is not None and y_vec.shape[1] != self._feature_map.num_parameters:
             raise ValueError("y_vec and class feature map have incompatible dimensions.\n" +
```

Comment:
Is there a reason why the blocks for `x_vec` and `y_vec` are slightly different?